mklove/modules/configure.libzstd

```diff
@@ -9,8 +9,12 @@
 #   mkl_check libzstd [<action>]
 #
 
+mkl_toggle_option "Feature" ENABLE_ZSTD "--enable-zstd" "Enable support for ZSTD compression" "y"
+
 function manual_checks {
-    local action=$1
+    local action=${1:-disable}
```

Comment:
why is a default value needed? it is always passed an action

Reply:
Not the PR author, but I assume @Whissi is simply following the precedent set by libssl and libsasl2:

  * https://github.com/edenhill/librdkafka/blob/b3223ecea638bfacc6d9ec981aa4df1d22ee9542/mklove/modules/configure.libssl#L22
  * https://github.com/edenhill/librdkafka/blob/b3223ecea638bfacc6d9ec981aa4df1d22ee9542/mklove/modules/configure.libsasl2#L17

If this is indeed unnecessary, perhaps it should be removed from those other modules as well.

Reply:
Yes, that was my intention: Keep current style.